src/client/components/app/ConnectWalletButton/index.tsx

```diff
@@ -34,6 +35,7 @@ export const ConnectWalletButton = ({ address, ensName, disabled, onClick }: Wal
 
   return (
     <StyledButton onClick={() => onClick && onClick()} disabled={disabled}>
+      {address && <Davatar size={18} address={address} style={{ marginRight: 6 }} generatedAvatarType="jazzicon" />}
```

Comment:
Remove inline styles and use styled components

```suggestion
      {address && <StyledDavatar size={18} address={address} generatedAvatarType="jazzicon" />}
```

and then do 
```
const StyledDavatar = styled(Davatar)`
    margin-right: .6rem;
`;
```